### PR TITLE
[next] Add option to file-name control to remove accent/diacritics

### PR DESCRIPTION
**Pull request by phuongnq**
### Ticket reference or full description of what's in the PR
An option to replace accent/diacritics to English characters from file-name control.
Ex:
Original string: à la carte, pied-à-terre, or crème
Transform string: a-la-carte--pied-a-terre--or-creme

### CODE REVIEW

**Review by sumerjabri — CHANGES_REQUESTED** [review]
Please see the comments on the support/3.1.x PR

File: static-assets/components/cstudio-common/resources/de/base.js [hunk 1]
```diff
@@ -845,6 +845,8 @@ CStudioAuthoring.Messages.registerBundle('contentTypes', 'de', {
   enableBrowseExisting: "Zeige 'Auswählen'",
   enableSearchExisting: "Zeige 'Suchen'",
   useSearch: 'Suche verwenden',
+  allowEditWithoutWarning: 'Allow Edit Without Warning',
+  replaceAccent: 'Convert accent/diacritics to latin characters',
```
Comment (rart) [inline static-assets/components/cstudio-common/resources/de/base.js]:
```suggestion
  replaceAccent: 'Konvertieren Sie Akzente/diakritische Zeichen in lateinische Zeichen',
```

File: static-assets/components/cstudio-common/resources/de/base.js [hunk 2]
```diff
@@ -845,6 +845,8 @@ CStudioAuthoring.Messages.registerBundle('contentTypes', 'de', {
   enableBrowseExisting: "Zeige 'Auswählen'",
   enableSearchExisting: "Zeige 'Suchen'",
   useSearch: 'Suche verwenden',
+  allowEditWithoutWarning: 'Allow Edit Without Warning',
```
Comment (rart) [inline static-assets/components/cstudio-common/resources/de/base.js]:
```suggestion
  allowEditWithoutWarning: 'Bearbeiten ohne Warnung zulassen',
```

File: static-assets/components/cstudio-common/resources/es/base.js [hunk 3]
```diff
@@ -811,6 +811,8 @@ CStudioAuthoring.Messages.registerBundle('contentTypes', 'es', {
   enableBrowseExisting: "Mostrar 'Seleccionar Existentes'",
   enableSearchExisting: "Mostrar 'Buscar Existentes'",
   useSearch: 'Usar Búsqueda',
+  allowEditWithoutWarning: 'Allow Edit Without Warning',
```
Comment (rart) [inline static-assets/components/cstudio-common/resources/es/base.js]:
```suggestion
  allowEditWithoutWarning: 'Permitir edición sin advertencia',
```

File: static-assets/components/cstudio-common/resources/es/base.js [hunk 4]
```diff
@@ -811,6 +811,8 @@ CStudioAuthoring.Messages.registerBundle('contentTypes', 'es', {
...
+  replaceAccent: 'Convert accent/diacritics to latin characters',
```
Comment (rart) [inline static-assets/components/cstudio-common/resources/es/base.js]:
```suggestion
  replaceAccent: 'Convertir acentos/diacríticos en caracteres latinos',
```

File: static-assets/components/cstudio-common/resources/kr/base.js [hunk 5]
```diff
@@ -778,6 +778,8 @@ CStudioAuthoring.Messages.registerBundle('contentTypes', 'kr', {
   enableBrowseExisting: '쇼 기존 항목 찾아보기',
   enableSearchExisting: '쇼 기존 검색',
   useSearch: '검색 사용',
+  allowEditWithoutWarning: 'Allow Edit Without Warning',
```
Comment (rart) [inline static-assets/components/cstudio-common/resources/kr/base.js]:
```suggestion
  allowEditWithoutWarning: '경고 없이 편집 허용',
```

File: static-assets/components/cstudio-common/resources/kr/base.js [hunk 6]
```diff
@@ -778,6 +778,8 @@ CStudioAuthoring.Messages.registerBundle('contentTypes', 'kr', {
...
+  replaceAccent: 'Convert accent/diacritics to latin characters',
```
Comment (rart) [inline static-assets/components/cstudio-common/resources/kr/base.js]:
```suggestion
  replaceAccent: '악센트/분음 부호를 라틴 문자로 변환',
```

**Review by rart — CHANGES_REQUESTED** [review]

**Review by rart — APPROVED** [review]

**Review by sumerjabri — CHANGES_REQUESTED** [review]
See comments in 3.1.x PR.

**Comment by rart** [comment]
Requirements were updated. Closing in favour of https://github.com/craftercms/studio-ui/pull/2761

**Pull request closed without merging**